dolphinscheduler-log-server/pom.xml

```diff
@@ -51,6 +51,16 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <configuration>
+                    <excludes>
+                        <exclude>*.yaml</exclude>
+                        <exclude>*.xml</exclude>
+                    </excludes>
+                </configuration>
+            </plugin>
```

Comment:
There is nothing under `dolphinscheduler-log-server/src/resources` so this plugin here is meaningless

Reply:
CI docker build process execution is abnormal, I try to add maven-jar-plugin plug-in, after adding, the test CI process can be executed normally.

```

...
[INFO] --- exec-maven-plugin:3.0.0:exec (docker-build) @ dolphinscheduler-log-server ---
#1 [internal] load build definition from Dockerfile
#1 sha256:fcec8f05b79051702a6f3f2ea66279662c2bd3cd2c6fa0a9f2f6a7881660b4c1
#1 transferring dockerfile: 37B done
#1 DONE 0.0s

#2 [internal] load .dockerignore
#2 sha256:78760670767d80dd9ab79cb2c95395857018839b80ff58d4e0f44f783375df26
#2 transferring context: 2B done
#2 DONE 0.0s

#3 [internal] load metadata for docker.io/library/openjdk:8-jre-slim-buster
#3 sha256:767c77441b83174da1b215130d37ccf6d9287199c440d9658a5d5555115def8c
#3 DONE 0.3s

#4 [1/4] FROM docker.io/library/openjdk:8-jre-slim-buster@sha256:0374c39c1f2fcadb5657a63a1fc181de3df3dcc608f8a88fd120e1d61b4def28
#4 sha256:b04ab0a0ef177ed12e96df045a6b844906fbede085f948442705eea367447236
#4 DONE 0.0s

#7 [internal] load build context
#7 sha256:d1bbad305d680d96b39263676248230a4f043668d2068f67507ad0ad99cf1e9f
#7 transferring context: 28B 0.0s done
#7 DONE 0.0s

#5 [2/4] RUN apt update ;     apt install -y curl ;     rm -rf /var/lib/apt/lists/*
#5 sha256:41384b81fe8a1c4796bc5ba9637e328d3feb0cd3ae6037716354cbea16d8aa37
#5 CACHED

#6 [3/4] WORKDIR /opt/dolphinscheduler
#6 sha256:e76937a8aa079f4654c8af0e4e39871bd3c52672f7ce53d4be6321a96ee8136b
#6 CACHED

#8 [4/4] ADD ./target/master-server /opt/dolphinscheduler
#8 sha256:ad589d36f9f072c98f0efdb5ce90157b36e62a63d1bc4f04092469849e2d2697
#8 ERROR: "/target/master-server" not found: not found
------
 > [4/4] ADD ./target/master-server /opt/dolphinscheduler:
------
failed to compute cache key: "/target/master-server" not found: not found
Error:  Command execution failed.
org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1)
    at org.apache.commons.exec.DefaultExecutor.executeInternal (DefaultExecutor.java:404)
    at org.apache.commons.exec.DefaultExecutor.execute (DefaultExecutor.java:166)
    at org.codehaus.mojo.exec.ExecMojo.executeCommandLine (ExecMojo.java:982)
    at org.codehaus.mojo.exec.ExecMojo.executeCommandLine (ExecMojo.java:929)
    at org.codehaus.mojo.exec.ExecMojo.execute (ExecMojo.java:457)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:39)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:122)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:61)
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for dolphinscheduler 2.0.0-SNAPSHOT:
[INFO] 
[INFO] dolphinscheduler ................................... SUCCESS [  2.562 s]
[INFO] dolphinscheduler-spi ............................... SUCCESS [  6.344 s]
[INFO] dolphinscheduler-alert ............................. SUCCESS [  0.139 s]
[INFO] dolphinscheduler-alert-api ......................... SUCCESS [  0.502 s]
[INFO] dolphinscheduler-alert-plugins ..................... SUCCESS [  0.224 s]
[INFO] dolphinscheduler-alert-email ....................... SUCCESS [  1.009 s]
[INFO] dolphinscheduler-alert-wechat ...................... SUCCESS [  0.731 s]
[INFO] dolphinscheduler-alert-dingtalk .................... SUCCESS [  0.560 s]
[INFO] dolphinscheduler-alert-script ...................... SUCCESS [  0.474 s]
[INFO] dolphinscheduler-alert-http ........................ SUCCESS [  0.547 s]
[INFO] dolphinscheduler-alert-feishu ...................... SUCCESS [  0.538 s]
[INFO] dolphinscheduler-alert-slack ....................... SUCCESS [  0.386 s]
[INFO] dolphinscheduler-common ............................ SUCCESS [  3.990 s]
[INFO] dolphinscheduler-remote ............................ SUCCESS [  2.741 s]
[INFO] dolphinscheduler-meter ............................. SUCCESS [  0.520 s]
[INFO] dolphinscheduler-dao ............................... SUCCESS [  3.718 s]
[INFO] dolphinscheduler-alert-server ...................... SUCCESS [ 17.802 s]
[INFO] dolphinscheduler-registry .......................... SUCCESS [  0.082 s]
[INFO] dolphinscheduler-registry-api ...................... SUCCESS [  0.323 s]
[INFO] dolphinscheduler-registry-plugins .................. SUCCESS [  0.115 s]
[INFO] dolphinscheduler-registry-zookeeper ................ SUCCESS [  0.766 s]
[INFO] dolphinscheduler-task-plugin ....................... SUCCESS [  0.074 s]
[INFO] dolphinscheduler-task-api .......................... SUCCESS [  0.965 s]
[INFO] dolphinscheduler-task-shell ........................ SUCCESS [  0.521 s]
[INFO] dolphinscheduler-datasource-plugin ................. SUCCESS [  0.090 s]
[INFO] dolphinscheduler-datasource-api .................... SUCCESS [  1.140 s]
[INFO] dolphinscheduler-datasource-clickhouse ............. SUCCESS [  0.316 s]
[INFO] dolphinscheduler-datasource-db2 .................... SUCCESS [  0.339 s]
[INFO] dolphinscheduler-datasource-hive ................... SUCCESS [  1.455 s]
[INFO] dolphinscheduler-datasource-mysql .................. SUCCESS [  0.287 s]
[INFO] dolphinscheduler-datasource-oracle ................. SUCCESS [  0.306 s]
[INFO] dolphinscheduler-datasource-postgresql ............. SUCCESS [  0.343 s]
[INFO] dolphinscheduler-datasource-sqlserver .............. SUCCESS [  0.473 s]
[INFO] dolphinscheduler-datasource-all .................... SUCCESS [  0.933 s]
[INFO] dolphinscheduler-task-datax ........................ SUCCESS [  1.156 s]
[INFO] dolphinscheduler-task-flink ........................ SUCCESS [  0.243 s]
[INFO] dolphinscheduler-task-http ......................... SUCCESS [  0.311 s]
[INFO] dolphinscheduler-task-mr ........................... SUCCESS [  0.274 s]
[INFO] dolphinscheduler-task-python ....................... SUCCESS [  0.390 s]
[INFO] dolphinscheduler-task-spark ........................ SUCCESS [  0.310 s]
[INFO] dolphinscheduler-task-sql .......................... SUCCESS [  0.816 s]
[INFO] dolphinscheduler-task-sqoop ........................ SUCCESS [  0.419 s]
[INFO] dolphinscheduler-task-procedure .................... SUCCESS [  1.015 s]
[INFO] dolphinscheduler-task-pigeon ....................... SUCCESS [  0.490 s]
[INFO] dolphinscheduler-ui ................................ SUCCESS [02:03 min]
[INFO] dolphinscheduler-service ........................... SUCCESS [  2.252 s]
[INFO] dolphinscheduler-server ............................ SUCCESS [  1.093 s]
[INFO] dolphinscheduler-api ............................... SUCCESS [ 14.236 s]
[INFO] dolphinscheduler-worker ............................ SUCCESS [ 10.817 s]
[INFO] dolphinscheduler-master ............................ SUCCESS [  7.559 s]
[INFO] dolphinscheduler-log-server ........................ FAILURE [  2.770 s]
[INFO] dolphinscheduler-python ............................ SKIPPED
[INFO] dolphinscheduler-standalone-server ................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  03:39 min
[INFO] Finished at: 2021-12-18T14:13:55Z
[INFO] ------------------------------------------------------------------------
Error:  Failed to execute goal org.codehaus.mojo:exec-maven-plugin:3.0.0:exec (docker-build) on project dolphinscheduler-log-server: Command execution failed. Process exited with an error: 1 (Exit value: 1) -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
Error:  
Error:  After correcting the problems, you can resume the build with the command
Error:    mvn <args> -rf :dolphinscheduler-log-server
Error: Process completed with exit code 1.


```

Reply:
The root cause is the Docker build is cached, I missed this in the first PR, can you please add `--no-cache` next to

https://github.com/apache/dolphinscheduler/blob/b350dc70c9cafa7d1a05a3329105c96417384f2c/pom.xml#L966

and 

https://github.com/apache/dolphinscheduler/blob/b350dc70c9cafa7d1a05a3329105c96417384f2c/pom.xml#L942

Adding `maven-jar-plugin` may work now but may not work in the future because it doesn't fix the root cause in fact

Reply:
See, this run doesn't work https://github.com/apache/dolphinscheduler/runs/4569793288?check_suite_focus=true

Reply:
Seems it still doesn't work, let me take some time to find out why.